src/pydata_sphinx_theme/assets/styles/content/_hacks.scss

```diff
@@ -9,8 +9,4 @@ span.linenos,
 div.doctest > div.highlight span.gp {
   /* gp: Generic.Prompt */
   user-select: none;
```

Comment:
I tested this locally on /examples/kitchen-sink/blocks.html#doctest-blocks. If I drag my mouse over the text and then paste it, it is pasted without the `>>>` prompt. I believe this preserved the intended behavior. (See https://github.com/pydata/pydata-sphinx-theme/pull/380.)